### PR TITLE
Replace `defaultValue` with `??`

### PR DESCRIPTION
**Pull request by dave-b-b**
# Description

This is a partial pull request to verify that I am making the right kinds of changes.

## Issue number and link

This is a partial fix. I don't want to link here and have this automatically close the ticket. But the ticket is #11674.

## Testing plan

I ran `npm run test`. It failed some of the tests, but none of the tests related to the changes that I made. (It seemed like they were all timeout issues)

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [ ] I have added my name to `CONTRIBUTORS.md`
- [ ] I have updated `CHANGES.md` with a short summary of my change
- [x] I have added or updated unit tests to ensure consistent code coverage
- [ ] I have updated the inline documentation, and included code examples where relevant
- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by jjspace** [comment]
> This is a partial fix. I don't want to link here and have this automatically close the ticket. But the ticket is 11674.

Just FYI, as long as you don't specifically use the exact [keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) that Github set up it won't automatically link a PR to an Issue.

File: packages/engine/Source/DataSources/BillboardGraphics.js [hunk 1]
```diff
@@ -303,7 +302,7 @@ Object.defineProperties(BillboardGraphics.prototype, {
    * @type {Property|undefined}
    */
   pixelOffsetScaleByDistance: createPropertyDescriptor(
-    "pixelOffsetScaleByDistance"
+    "pixelOffsetScaleByDistance",
```
Comment (jjspace) [inline packages/engine/Source/DataSources/BillboardGraphics.js:305]: It seems like you have the wrong version of `prettier` installed or your editor configuration is wrong. These changes are distracting from the intended change of this PR (I'm working on the update in #12206). Can you do a `npm install` and make sure to run `npm run prettier`? Also probably worth checking that the pre-commit hook is working as intended so these can't even be committed.

Comment (dave-b-b) [reply packages/engine/Source/DataSources/BillboardGraphics.js]: Okay. I see the pre-commit hook. There was probably something I did wrong. But it looks like it is preventing commits now. 

I ran prettier and merged the changes into my pull request.

image

File: packages/widgets/Source/Viewer/Viewer.js [hunk 2]
```diff
@@ -2265,7 +2256,7 @@ function updateZoomTarget(viewer) {
 
   const scene = viewer.scene;
   const camera = scene.camera;
-  const zoomOptions = defaultValue(viewer._zoomOptions, {});
+  const zoomOptions = viewer._zoomOptions ?? defaultValue.EMPTY_OBJECT;
```
Comment (jjspace) [inline packages/widgets/Source/Viewer/Viewer.js:2259]: The `defaultValue.EMPTY_OBJECT` is [frozen](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) and cannot be modified. Most of the places we use it are only _reading_ information from the object so this is not a problem. However some places we used `defaultValue` we do modify the properties later (like this one). There's probably discussion to be had about whether these objects _should_ be modified for later but for the sake of keeping this PR isolated let's just preserve the existing behavior.
Basically don't replace calls like these with the `EMPTY_OBJECT`, just keep the `{}`. I think this is what's causing your test failures.

**Review by jjspace — COMMENTED** [review]
Thanks @dave-b-b, looks like you're on the right track! I think all the existing replacements look good (other than the comment below). Do you want to continue with the rest in this PR and we can merge them all at once?

File: packages/widgets/Source/Viewer/Viewer.js (hunk 2 above)
Comment (dave-b-b) [reply packages/widgets/Source/Viewer/Viewer.js:2259]: @jjspace I'll keep working on this so we can merge everything at once. Thanks for working with me! And sorry for the late response.

**Comment by jjspace** [comment]
@dave-b-b any progress on the rest of the replacements?

Also please follow these [instructions](https://github.com/CesiumGS/cesium/pull/12206#issuecomment-2377930855) to merge in recent prettier changes

**Comment by ggetz** [comment]
Hi @dave-b-b, thank you for the work on this so far!

There's a [related PR open which could be dependent on this one]( https://github.com/CesiumGS/cesium/pull/12238#issuecomment-2402729851). Would you mind letting us know your expected timeline for working on this PR? We just want to figure out what makes the most sense logistically. 😃

**Comment by dave-b-b** [comment]
@ggetz Yes, sorry for the delay. I saw the related PR. I will have this done no later than the end of the week. But I'm going to try to get this done today.

**Comment by jjspace** [comment]
Closing in favor of https://github.com/CesiumGS/cesium/pull/12252
